app/src/main/res/layout/promoted_story_card.xml

```diff
@@ -63,7 +63,7 @@
           android:fontFamily="sans-serif-medium"
```

Comment:
Instead of `bottom_rounded_rect_white_background` use `component_color_shared_rounded_rect_background_color`

Reply:
And can we delete `bottom_rounded_rect_white_background ` as it not use anywhere instead of this file.

Reply:
Yes you can delete it if it is not getting used.

Reply:
Done.